docs/deciding_what_data_to_stream_with_dfe_analytics.md

```diff
@@ -0,0 +1,70 @@
+
+## How do we exclude data from streaming?
+Data for which DfE has no basis to store or process **must not** be streamed into BigQuery by ```dfe-analytics```.
+
+This data may take the form of individual fields within a table in an application's database, or entire tables.
+
+To exclude these fields, remove them from ```config/analytics.yml``` and add them to ```config/analytics_blocklist.yml```.
+
+For more information see [here](https://github.com/DFE-Digital/dfe-analytics/tree/SL-dev?tab=readme-ov-file#5-send-database-events).
+
+## How long should we keep data for?
+Data processed under the 'public task' basis for processing described in the previous section may continue to be stored and processed for as long as it is needed for any of the public task(s) that is stored for.
+
+This means that once it is no longer useful for any such task it **must** be deleted. For example:
+- Data to support analysis of teachers' progression throughout their careers so that the effectiveness of policy measures to retain teachers can be evaluated might need to be retained for 50 years
+- Data that is only stored to support evaluation of the impact of policy changes on ITT recruitment might need to be stored for a number of years (say, 7) but not longer
+- Data that is only stored to support evaluation of the immediate effects of a design or content change to a digital service might only need to be stored for around a year
+
+IAOs and their teams should also consider public tasks which may need to be performed by other teams within DfE, both now and in the future, when evaluating how long data should be retained for.
+
+Teams **should** consider [implementing a data retention schedule](https://github.com/DFE-Digital/dfe-analytics-dataform/?tab=readme-ov-file#data-retention-schedules) in ```dfe-analytics-dataform``` to ensure that data streamed by ```dfe-analytics``` is deleted automatically once it is no longer required.
+
+## How do we hide more sensitive data when it is streamed?
+Data that is more sensitive than most personal data **should** be marked as such when it is streamed into BigQuery by ```dfe-analytics``` - if it is streamed at all. This includes:
+- Human-readable PII, such as names, email addresses, home addresses and TRNs
+- [Special category data](https://ico.org.uk/for-organisations/uk-gdpr-guidance-and-resources/lawful-basis/special-category-data/what-is-special-category-data/) under GDPR
+
+This data should be both:
+1. hidden when it is streamed using ```dfe-analytics``` [hidden field functionality](https://github.com/DFE-Digital/dfe-analytics/tree/SL-dev?tab=readme-ov-file#5-send-database-events)
+2. hidden when it is transformed using ```dfe-analytics-dataform``` and Dataform as described [here](https://github.com/DFE-Digital/dfe-analytics-dataform/?tab=readme-ov-file#hidden-fields).
```

Comment:
Suggest we include the types of data here (pasted below), rather than linking it. Unless we expect that this data could change and it is best to link to the most up to date version?

personal data revealing racial or ethnic origin;
personal data revealing political opinions;
personal data revealing religious or philosophical beliefs;
personal data revealing trade union membership;
genetic data;
biometric data (where used for identification purposes);
data concerning health;
data concerning a person’s sex life; and
data concerning a person’s sexual orientation.

Reply:
Added